pytorch_lightning/loops/epoch/evaluation_epoch_loop.py

```diff
@@ -112,8 +108,8 @@ def advance(
 
         self.batch_progress.increment_processed()
 
-        # hook + store predictions
-        self.on_evaluation_batch_end(output, batch, batch_idx, dataloader_idx)
+        # track loss history
```

Comment:
You accidentally removed this

```suggestion
        self.on_evaluation_batch_end(output, batch, batch_idx, dataloader_idx)

        # track loss history
```